README.md

```diff
@@ -1,46 +1,53 @@
-# unc0ver
+# Unc0ver
 ### The most advanced jailbreak tool
-![unc0ver logo](https://raw.githubusercontent.com/pwn20wndstuff/Undecimus/master/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png?token=AlyO4xDujoguob2DCFfUbNI8jO82OyCgks5bx5ZPwA%3D%3D)
+![unc0ver logo](/Undecimus/Assets.xcassets/AppIcon.appiconset/Icon-App-60x60%403x.png)
 
-unc0ver jailbreak for iOS 11.0 - 11.4b3<br/>
+unc0ver jailbreak for iOS 11.0 - 11.4.1<br/>
 by [@pwn20wnd](https://twitter.com/Pwn20wnd) & [@sbingner](https://twitter.com/sbingner)<br/>
 UI by [@DennisBednarz](https://twitter.com/DennisBednarz) & [Samg_is_a_Ninja](https://reddit.com/u/Samg_is_a_Ninja)<br/>
 
 ## The most outstanding changes over the other jailbreaks
-* All exploits in same app
-* Detailed error messages
+* All exploits in the same app
+* Detailed  error messages
```

Comment:
Was it necessary to add an extra space here?

Reply:
fixed